substrate/frame/salary/src/tests/integration.rs

```diff
@@ -19,24 +19,23 @@
 
 use crate as pallet_salary;
 use crate::*;
-use frame_support::{
-	assert_noop, assert_ok, derive_impl, hypothetically,
-	pallet_prelude::Weight,
-	parameter_types,
-	traits::{ConstU64, EitherOf, MapSuccess, NoOpPoll},
-};
+
+use frame::{prelude::*, runtime::prelude::*, testing_prelude::*,
+			traits::{ConstU64, EitherOf, MapSuccess, NoOpPoll,
+					 Convert, ReduceBy, ReplaceWithDefault},
+			deps::{
+				sp_io::{self, MultiRemovalResults},
+				sp_runtime,
+			}};
 use pallet_ranked_collective::{EnsureRanked, Geometric};
 use sp_core::{ConstU16, Get};
-use sp_runtime::{
-	traits::{Convert, ReduceBy, ReplaceWithDefault},
-	BuildStorage,
-};
+use crate::tests::integration::sp_api_hidden_includes_construct_runtime::hidden_include::hypothetically;
```

Comment:
This import was originally from `frame_support`. Why not -

```suggestion
use frame::deps::frame_support::hypothetically;
```

or add it to one of the preludes where it makes sense?